Fix crash on opening Discus algorithm dialog (#33973)
mantidproject/mantid: docs/source/release/v6.4.0/Framework/Algorithms/Bugfixes/33973_FixCrashOnOpeningDiscusAlgDialog.rst

> - fix crash when opening the algorithm dialog for :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` if a group workspace is present in the Analysis Data Service

Reviewer: Just so Sarah and I don't have to update it later
```suggestion
- Workbench will no longer crash if the algorithm dialog for :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` is opened while a ``GroupWorkspace`` is present in the Workspaces list.
```
